ui/src/app/components/runtime-configuration-panel/confirm-environment-delete-with-pd.tsx

```diff
@@ -1,29 +1,48 @@
 import * as React from 'react';
 
+import { Disk } from 'generated/fetch';
+
+import { UIAppType } from 'app/components/apps-panel/utils';
 import { Button } from 'app/components/buttons';
 import { FlexRow } from 'app/components/flex';
 import { ClrIcon } from 'app/components/icons';
 import { RadioButton } from 'app/components/inputs';
 import colors from 'app/styles/colors';
-import { ComputeType, detachableDiskPricePerMonth } from 'app/utils/machines';
+import { DEFAULT, switchCase } from 'app/utils';
+import { detachableDiskPricePerMonth } from 'app/utils/machines';
 import { formatUsd } from 'app/utils/numbers';
-import { RuntimeStatusRequest } from 'app/utils/runtime-utils';
 
 import { BackupFilesHelpSection } from './backup-files-help-section';
 import { styles } from './styles';
 
-const { useState, Fragment } = React;
+const { useState } = React;
+
+interface ConfirmDeleteEnvironmentWithPDProps {
+  onCancel: () => void;
+  onConfirm: (deletePDSelected: boolean) => Promise<void>;
+  appType: UIAppType;
+  // assumption: usingDataproc is always false for GKE apps
+  usingDataproc: boolean;
+  disk: Disk;
+}
 
-export const ConfirmDeleteRuntimeWithPD = ({
+export const ConfirmDeleteEnvironmentWithPD = ({
   onCancel,
   onConfirm,
-  computeType,
+  appType,
+  usingDataproc,
   disk,
-}) => {
+}: ConfirmDeleteEnvironmentWithPDProps) => {
   const [deleting, setDeleting] = useState(false);
-  const [runtimeStatusReq, setRuntimeStatusReq] = useState(
-    RuntimeStatusRequest.DeleteRuntime
+  const [deletePDSelected, setDeletePDSelected] = useState(false);
+
+  const volumeHome = switchCase(
+    appType,
+    [UIAppType.JUPYTER, () => '/home/jupyter'],
```

Comment:
surprised to see this is the only place this path appears, but it seems to be so